cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/steps/LoadStep/help.html

```diff
@@ -0,0 +1,17 @@
+<div>
+    Takes a filename in the workspace and runs it as Groovy source text.
+    <p>
+        The loaded file can contain statements at top level or just load and run a closure. For example:
```

Comment:
This is getting into the grey area of longer content, overlapping that described more thoroughly [in the “tutorial”](https://github.com/jenkinsci/workflow-plugin/blob/master/TUTORIAL.md#manual-loading). Cf. #203. I tend to recommend the full content be included here in `help.html`, where it will be available in other media such as via [JENKINS-26126](https://issues.jenkins-ci.org/browse/JENKINS-26126).